[0m15:29:27.449112 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f5142b89970>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f5140f5d550>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f51416f00b0>]}


============================== 15:29:27.451638 | 32b82cba-be07-4329-a9b0-77b37e9344a5 ==============================
[0m15:29:27.451638 [info ] [MainThread]: Running with dbt=1.9.4
[0m15:29:27.451999 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/home/stitch/.dbt', 'debug': 'False', 'version_check': 'True', 'log_path': 'logs', 'warn_error': 'None', 'fail_fast': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'log_format': 'default', 'invocation_command': 'dbt init altera-data-platform', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'target_path': 'None', 'static_parser': 'True', 'send_anonymous_usage_stats': 'True'}
[0m15:29:27.464059 [warn ] [MainThread]: [ConfigFolderDirectory]: Unable to parse logging event dictionary. Failed to parse dir field: expected string or bytes-like object, got 'PosixPath'.. Dictionary: {'dir': PosixPath('/home/stitch/.dbt')}
[0m15:29:27.464354 [info ] [MainThread]: Creating dbt configuration folder at 
[0m15:29:51.388302 [debug] [MainThread]: Starter project path: /home/stitch/engineering-demo/venv/lib/python3.12/site-packages/dbt/include/starter_project
[0m15:29:51.397089 [info ] [MainThread]: 
Your new dbt project "altera_data_platform" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m15:29:51.397608 [info ] [MainThread]: Setting up your profile.
[0m15:31:11.473002 [info ] [MainThread]: Profile altera_data_platform written to /home/stitch/.dbt/profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m15:31:11.475228 [debug] [MainThread]: Resource report: {"command_name": "init", "command_success": true, "command_wall_clock_time": 95.68724, "process_in_blocks": "63736", "process_kernel_time": 0.097288, "process_mem_max_rss": "98192", "process_out_blocks": "88", "process_user_time": 0.816044}
[0m15:31:11.475558 [debug] [MainThread]: Command `dbt init` succeeded at 15:31:11.475485 after 95.69 seconds
[0m15:31:11.475801 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f51413857f0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f5140b11970>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f51410377a0>]}
[0m15:31:11.476052 [debug] [MainThread]: Flushing usage events
[0m15:31:11.665005 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m09:23:25.350594 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fcc2a5acec0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fcc285d25d0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fcc285d10d0>]}


============================== 09:23:25.357379 | 9ca1dbbf-5d33-4262-83e6-339714256a5f ==============================
[0m09:23:25.357379 [info ] [MainThread]: Running with dbt=1.9.4
[0m09:23:25.358679 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/home/stitch/.dbt', 'debug': 'False', 'version_check': 'True', 'log_path': 'logs', 'fail_fast': 'False', 'warn_error': 'None', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'log_format': 'default', 'invocation_command': 'dbt deps', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'static_parser': 'True', 'target_path': 'None', 'introspect': 'True', 'send_anonymous_usage_stats': 'True'}
[0m09:23:25.359733 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path /home/stitch/engineering-demo/dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m09:23:25.361356 [debug] [MainThread]: Resource report: {"command_name": "deps", "command_success": false, "command_wall_clock_time": 0.087821856, "process_in_blocks": "46160", "process_kernel_time": 0.357516, "process_mem_max_rss": "94464", "process_out_blocks": "16", "process_user_time": 1.685434}
[0m09:23:25.362103 [debug] [MainThread]: Command `dbt deps` failed at 09:23:25.361924 after 0.09 seconds
[0m09:23:25.362698 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fcc2a5acec0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fcc280a1280>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fcc285d10d0>]}
[0m09:23:25.363337 [debug] [MainThread]: Flushing usage events
[0m09:23:25.581300 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m10:02:55.243107 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7efc67af8da0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7efc691c4d40>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7efc68503ad0>]}


============================== 10:02:55.245156 | 83381f49-5f61-46a8-986d-a694f97aa068 ==============================
[0m10:02:55.245156 [info ] [MainThread]: Running with dbt=1.9.4
[0m10:02:55.245507 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/home/stitch/.dbt', 'fail_fast': 'False', 'warn_error': 'None', 'log_path': 'logs', 'version_check': 'True', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'log_format': 'default', 'static_parser': 'True', 'invocation_command': 'dbt debug', 'introspect': 'True', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'send_anonymous_usage_stats': 'True'}
[0m10:02:55.252722 [info ] [MainThread]: dbt version: 1.9.4
[0m10:02:55.253013 [info ] [MainThread]: python version: 3.12.3
[0m10:02:55.253221 [info ] [MainThread]: python path: /home/stitch/engineering-demo/venv/bin/python3
[0m10:02:55.253414 [info ] [MainThread]: os info: Linux-5.15.167.4-microsoft-standard-WSL2-x86_64-with-glibc2.39
[0m10:02:55.255874 [error] [MainThread]: Encountered an error:
Parsing Error
  Env var required but not provided: 'POSTGRES_HOST'
[0m10:02:55.256439 [debug] [MainThread]: Resource report: {"command_name": "debug", "command_success": false, "command_wall_clock_time": 0.04589811, "process_in_blocks": "8", "process_kernel_time": 0.087148, "process_mem_max_rss": "96424", "process_out_blocks": "8", "process_user_time": 0.735922}
[0m10:02:55.256703 [debug] [MainThread]: Command `dbt debug` failed at 10:02:55.256639 after 0.05 seconds
[0m10:02:55.256921 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7efc67f901d0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7efc6801a2a0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7efc67bb4fb0>]}
[0m10:02:55.257184 [debug] [MainThread]: Flushing usage events
[0m10:02:54.431728 [debug] [MainThread]: An error was encountered while trying to flush usage events
